tests/src/Functional/OgSelectionWidgetTest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Drupal\Tests\og\Functional;
+
+use Drupal\entity_test\Entity\EntityTestBundle;
+use Drupal\entity_test\Entity\EntityTestWithBundle;
+use Drupal\node\Entity\Node;
+use Drupal\og\Og;
+use Drupal\og\OgGroupAudienceHelperInterface;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\node\Traits\ContentTypeCreationTrait;
+use Drupal\simpletest\NodeCreationTrait;
```

Comment:
Deprecated Drupal\simpletest\NodeCreationTrait, see https://www.drupal.org/node/2884454
```suggestion
use Drupal\Tests\node\Traits\NodeCreationTrait;
```